docs/docs/getting-started/client.md

```diff
@@ -449,7 +449,24 @@ export async function getServerSideProps(ctx) {
 }
 ```
 
-If every one of your pages needs to be protected, you can do this in `getInitialProps` in `_app`, otherwise you can do it on a page-by-page basis. Alternatively, you can do per page authentication checks client side, instead of having each authentication check be blocking (SSR) by using the method described below in [alternative client session handling](#custom-client-session-handling).
+If every one of your pages needs to be protected, you can do this in `getInitialProps` in `_app`, otherwise you can do it on a page-by-page basis. 
+
+```jsx title="pages/_app.js"
+import type { AppContext, AppInitialProps } from "next/app";
+import App from "next/app";
+import { getSession } from "next-auth/react";
+import { Session } from "next-auth";
+
```

Comment:
Maybe put the whole 

```

export default function App({
  Component,
  pageProps: { session, ...pageProps },
}) {
  return (
    // `session` comes from `getServerSideProps` or `getInitialProps`.
    // Avoids flickering/session loading on first load.
    <SessionProvider session={session} refetchInterval={5 * 60}>
      <Component {...pageProps} />
    </SessionProvider>
  )
}
```

Part back into here to make it more straightforward to follow?